platform/remote/Makefile.remote

```diff
@@ -36,11 +36,18 @@ MODULES += core/net core/net/mac \
            core/net/mac/contikimac \
            core/net/llsec
 
+PYTHON = python
+BSL_FLAGS += -e -w -v -a $$($(OBJDUMP) -h $*.elf | grep -B1 LOAD | grep -Ev 'LOAD|\-\-' | awk '{print "0x" $$5}' | sort -g | head -1)
+
+ifdef PORT
+  BSL_FLAGS += -p $(PORT)
+endif
+
```

Comment:
There is already a block that does all that near line 7 of this file, so with this change we define `PYTHON`, `BSL_FLAGS` etc twice